docs/articles/documentation/using-testcafe/installing-testcafe.md

```diff
@@ -81,3 +81,13 @@ Use `npx testcafe` if you need to quickly run TestCafe from a directory outside
 ```sh
 npx testcafe chrome tests/
 ```
+
+## Screen Recording Permission
+
+TestCafe requires screen recording permission on macOS (v10.15 Catalina and newer) to perform test actions, take screenshots and record videos. When TestCafe starts the first time, macOS asks you to allow screen recording for *TestCafe Browser Tools*.
+
+![Screen Recording Permissions](../../images/recording-permissions.png)
+
+Click **Open System Preferences** and check **TestCafe Browser Tools** to grant the permission.
+
+When you update TestCafe, macOS may reset its security permissions (it depends on which TestCafe components are updated). In this case, the system repeats the request the next time you run tests.
```

Comment:
The problem was that the system kept spawning consent dialogs despite the mark in the "Privacy & Security" Settings was ticked. Users were confused and couldn't understand that they had to clear and re-tick the mark.

Reply:
Also, I feel that attaching a screenshot of the Privacy & Security panel is required:

![img](https://dropshare.zendesk.com/hc/article_attachments/360042956714/Screenshot_2019-08-04_at_20.18.01.png)

Reply:
ping @VasilyStrelyaev